src/game-support/gta-4.md

```diff
@@ -0,0 +1,13 @@
+# Grand Theft Auto IV
+<!-- script:Aliases [
+    "GTA 4",
+    "GTA IV",
+    "Grand Theft Auto 4"
+] -->
+
+{{#template ../templates/rating.md status=Garbage installs=Yes opens=No}}
+
+> [!WARNING]
+> The Rockstar Games Launcher will fail with the `The Rockstar Games Launcher failed to initialize!` error when `DXVK` is turned `off` in your `Bottle Configuration`. `DXVK` can turned `on` to workaround the launcher error.
+
+Crashes during load without error message and is therefore currently unplayable in Whisky.
```

Comment:
Does the game crashes itself or the Rockstar Launcher?

Reply:
Additionally please add Steam ID of GTA IV as it may be helpful for others to locate this game on Steam.